deployment/fedora-package-x64/pkg-src/jellyfin.spec

```diff
@@ -38,7 +38,7 @@ Requires:       libcurl, fontconfig, freetype, openssl, glibc libicu
 # Requirements not packaged in main repos
 # COPR @dotnet-sig/dotnet or
 # https://packages.microsoft.com/rhel/7/prod/
-BuildRequires:  dotnet-runtime-3.0, dotnet-sdk-3.0
+BuildRequires:  dotnet-runtime-3.0, dotnet-sdk-3.1
```

Comment:
Shouldn’t this be runtime 3.1?

Reply:
Good catch, I fixed it and squashed the commit.